Python/codegen.c

```diff
@@ -4164,6 +4164,7 @@ codegen_sync_comprehension_generator(compiler *c, location loc,
 
     if (IS_JUMP_TARGET_LABEL(start)) {
         depth++;
+        ADDOP(c, LOC(gen->iter), CHECK_ITER);
```

Comment:
What if we emit `GET_ITER` here, relying on iterators obeying `iter(self) is self`? That would allow us to fix the crash without adding a new bytecode?

Reply:
I think so. It will fix crash without changing magic number and bytecode values. Overhead with GET_ITER seems to be very little.